storage_service/locations/models/package.py

```diff
@@ -437,6 +438,9 @@ def store_aip(self, origin_location, origin_path):
         # Update package status once transferred to SS
         if dest_space.access_protocol not in (Space.LOM, Space.ARKIVUM):
             self.status = Package.UPLOADED
+        if related_package_uuid is not None:
+            related_package = Package.objects.get(uuid=related_package_uuid)
+            self.related_packages.add(related_package)
```

Comment:
I've also got an Archivematica branch that deals with this possibility in the storeAIP microservice... here's the commit:

https://github.com/artefactual/archivematica/commit/e0455b97d8422adf67c36ecbec94f4a7f630daf2